### DISCUSSION TITLE
为什么没有将基础组件分离？

### DISCUSSION THREAD

**Discussion by jin2gh** [ant-design/ant-design-mobile#3939] [Q&A] [answered]
最新的 v5 版本，为什么没有采用 antd 的方式，将基础组件分离出来，然后引用基础组件？

**Comment by awmleer** [comment, marked as answer]
其实主要还是从可维护性的角度考虑，基础组件如果分离出一个个独立的 npm 包，会极大地增加维护成本和难度。

然后，不如反方向地想一下，把基础组件分离，会带来什么收益么？对目前的 antd-mobile 来说，似乎也没什么特别大的价值……
